Detectors/TPC/simulation/include/TPCSimulation/BoxCluster.h

```diff
@@ -53,7 +53,7 @@ namespace o2{
 		 Short_t size);
 
       /// Destructor
-      ~BoxCluster() override = default;
+      ~BoxCluster() = default;
```

Comment:
Just for curiosity, why was the `override` removed?

Reply:
This is because clusters are no longer polymorphic (in a restricted sense). They don't need a virtual table. I will never say `Cluster* cl; delete cl;` when cl is actually of type `BoxCluster`.